fdb-relational-grpc/src/test/java/com/apple/foundationdb/relational/jdbc/ProtobufConversionTest.java

```diff
@@ -63,4 +72,37 @@ public void testToColumnIntegerBiFunction() {
         column = TypeConversion.toColumn(a, biFunction);
         Assertions.assertEquals(a, column.getInteger());
     }
+
+    @Test
+    void testResultSetWithContinuation() throws Exception {
+        MockContinuation continuation = new MockContinuation(Continuation.Reason.TRANSACTION_LIMIT_REACHED, null, false, false);
+        RelationalResultSet resultSet = TestUtils.resultSet(
+                "TestType",
+                List.of(Types.INTEGER, Types.INTEGER, Types.INTEGER),
+                continuation,
+                TestUtils.row(1, 2, 3), TestUtils.row(4, 5, 6));
+        ResultSet converted = TypeConversion.toProtobuf(resultSet);
+
+        Assertions.assertEquals(2, converted.getRowCount());
+        assertRow(List.of(1, 2, 3), converted.getRow(0));
+        assertRow(List.of(4, 5, 6), converted.getRow(1));
+        assertContinuation(continuation.serialize(), RpcContinuationReason.TRANSACTION_LIMIT_REACHED, converted.getContinuation());
+    }
+
+    private void assertContinuation(byte[] state, RpcContinuationReason reason, RpcContinuation actual) {
+        if (state == null) {
+            Assertions.assertNull(actual.getInternalState().toByteArray());
```

Comment:
You don't actually utilize this path in this class

Reply:
Right - that remained from an earlier version of the test. Changed.